game-core/src/main/java/games/strategy/triplea/ui/MapPanel.java

```diff
@@ -651,14 +650,12 @@ private void updateUndrawnTiles(final List<Tile> undrawnTiles, final int preDraw
     }
   }
 
-  private void drawTiles(final Graphics2D g, final List<Tile> images, final GameData data, Rectangle2D.Double bounds,
-      final List<Tile> undrawn) {
-    final List<Tile> tileList = tileManager.getTiles(bounds);
-    bounds = new Rectangle2D.Double(bounds.getX(), bounds.getY(), bounds.getHeight(), bounds.getWidth());
```

Comment:
Little confused on this line. Any insight?

Reply:
It seems like the newly created rectangle just switched the height and width of the original rectangle. 
However the height and width were never queried so this line effectively did nothing.